### PR TITLE
feat: add stakeholders

### PR DESCRIPTION
**Pull request by Aashish-Upadhyay-101**
PR: #137

### CODE REVIEW

**Comment by Aashish-Upadhyay-101** [comment]
@dahal dai, @chetannn dai
This PR is now ready for review!

File: prisma/schema.prisma [hunk 1]
```diff
@@ -158,7 +158,7 @@ model Stakeholder {
   name                String
   email               String
   institutionName     String?
-  stakeholder_type    StakeholderTypeEnum         @default(INDIVIDUAL)
```
Comment (dahal) [inline prisma/schema.prisma:161]: 💜

File: src/components/stakeholder/stakeholder-uploader.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+  return (
+    <div className="space-y-4">
+      <div className="text-sm leading-6 text-neutral-600">
+        Please download the{" "}
+        <Link
+          href="/sample-csv/opencap-stakeholders-template.csv"
+          target="_blank"
+          rel="noopener noreferrer"
+          className="rounded bg-gray-300/70 px-2 py-1 text-xs font-medium hover:bg-gray-400/50"
```
Comment (dahal) [inline src/components/stakeholder/stakeholder-uploader.tsx:72]: We prolly need a `download` attribute

```
<Link
  download
  href="/sample-csv/opencap-stakeholders-template.csv"
  target="_blank"
  ...

File: src/trpc/routers/stakeholder-router/procedures/add-stakeholders.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+              existingStakeholder.email === stakeholder.email,
+          );
+        },
+      );
+
+      // bulk create
+      if (uniqueStakeholders.length > 0) {
+        await ctx.db.stakeholder.createMany({
+          data: uniqueStakeholders,
+          skipDuplicates: true,
```
Comment (dahal) [inline src/trpc/routers/stakeholder-router/procedures/add-stakeholders.ts:40]: Do we need to pre-filter if we have `skipDuplicates`?

**Review by dahal — CHANGES_REQUESTED** [review]
Its looking great @Aashish-Upadhyay-101 , I have left few comments. Please address those and it'll be ready to merge 🚀

File: src/trpc/routers/stakeholder-router/procedures/add-stakeholders.ts (hunk 3 above)
Comment (Aashish-Upadhyay-101) [reply src/trpc/routers/stakeholder-router/procedures/add-stakeholders.ts:40]: oh yeah, even if I wrote the `skipDuplicates: true` it was still creating duplicate entries, so I wrote my own filtering.
now that I just released that the `email` field in the schema should be `@unique` which wasn't before.
I added a unique constraint in the email field in the schema and now it is working 👍 

Thanks !

**Review by dahal — APPROVED** [review]

**Pull request merged**